rerun_cpp/CMakeLists.txt

```diff
@@ -57,8 +57,13 @@ if(NOT TARGET rerun_c)
             message(WARNING "Unknown architecture ${CMAKE_SYSTEM_PROCESSOR}, can't find rerun_c library.")
         endif()
     elseif(UNIX) # if(LINUX) # CMake 3.25
-        # TODO(andreas): Arm support.
-        set(RERUN_C_LIB_DEFAULT ${RERUN_C_DEFAULT_LIB_DIR}/librerun_c__linux_x64.a)
+        if("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "x86_64")
+            set(RERUN_C_LIB_DEFAULT ${RERUN_C_DEFAULT_LIB_DIR}/librerun_c__linux_x64.a)
+        elseif("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "arm64")
```

Comment:
I get an unknown "aarch64" architecture error. Half of the world call that arm64, the other aarch64. We must check all inputs against both versions imo.